cp-ksqldb-cli/Dockerfile.ubi8

```diff
@@ -52,10 +52,28 @@ name=Confluent repository \n\
 baseurl=${CONFLUENT_PACKAGES_REPO}/ \n\
 gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
-enabled=1 " > /etc/yum.repos.d/confluent.repo \
-    && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
-    && echo "===> Cleaning up ..."  \
+enabled=1 " > /etc/yum.repos.d/confluent.repo
+
+RUN echo "===> Installing Confluent Hub client ${CONFLUENT_VERSION}..." \
+    && yum install -y confluent-hub-client-${CONFLUENT_VERSION}
+
+RUN echo "===> HACK to make cc-role-definitions available in docker image..." \
+    && yum install -y yum-utils \
+    && yum install -y cpio \
+    && cd /tmp \
+    && yumdownloader confluent-ksqldb --resolve \
+    && CONFLUENT_KSQLDB_RPM=$(ls /tmp/confluent-ksqldb-*.rpm) \
+    && echo "===> Extracting the rpm ${CONFLUENT_KSQLDB_RPM} to get the cc-role-definitions jar" \
+    && mkdir /tmp/extracted-ksqldb \
+    && cd /tmp/extracted-ksqldb \
+    && rpm2cpio ${CONFLUENT_KSQLDB_RPM} | cpio -idmv
+
+RUN cp /tmp/extracted-ksqldb/usr/share/java/ksqldb/cc-role-definitions*.jar /usr/share/java/${COMPONENT}/ \
```

Comment:
This can be integrated in the previous RUN command itself. Running `rm -rf <rpm> <tmp_dir>` will not reduce the size of docker image as that is present in the previous layer